haralyzer/multihar.py

```diff
@@ -126,6 +126,7 @@ def js_load_time(self):
         Returns aggregate javascript load time.
         """
         load_times = self.get_load_times('js')
+        print(load_times)
```

Comment:
Looks like some debug stuff that needs to be removed.

Reply:
Lol whoops